### DISCUSSION TITLE
Open multiple admin views without navigation through menu again?

### DISCUSSION THREAD

**Discussion by CodeBrauer** [pimcore/pimcore#12665] [Q&A / Support] [answered]
Hello to all!

I have a newbie question: Is it possible to open multiple pages/modules in new tabs (pimcore not browser-tabs) in the admin interface with a hotkey+click or (mouse wheel click by default)? 

For example: As a user, I want to open "Users" and "Roles" at the same time without navigating through the menu again.

**Comment by kingjia90** [comment, marked as answer]
You can set and use Hotkey, they are under `Key bindings` tab on a specific user. If the tab is already open, it will switch to it.
Try `CTRL+ ALT + U` which is the default for `Users` 😄 

![image](https://user-images.githubusercontent.com/6014195/179232876-6b57fc5a-4258-4df5-bbef-bebe38054f61.png)
